main.go

```diff
@@ -174,7 +174,7 @@ func tr1d1um(arguments []string) (exitCode int) {
 		}
 		webhookConfig.Argus.HTTPClient = newHTTPClient(argusClientTimeout, tracing)
 
-		svc, stopWatch, err := ancla.Initialize(webhookConfig, getLogger)
+		svc, stopWatch, err := ancla.Initialize(webhookConfig, getLogger, logging.WithLogger)
```

Comment:
Same question as in here: https://github.com/xmidt-org/caduceus/pull/271/files#r668334418

Reply:
Answered in caduceus thread - thx!